src/open_inwoner/cms/cases/views/status.py

```diff
@@ -507,6 +501,11 @@ def register_by_email(self, form, recipient_email):
     def register_by_api(self, form, config: OpenKlantConfig):
         assert config.has_api_configuration()
 
+        try:
+            ztc = ZaakTypeConfig.objects.filter_case_type(self.case.zaaktype).get()
+        except ObjectDoesNotExist:
```

Comment:
Does `filter_case_type` always return a single object?

Reply:
It does. It returns a queryset so it can be chained with other filters and situations that want a queryset (eg: `filter_enabled_for_case_type()` and `filter_questions_enabled_for_case_type()`)